### PR TITLE
Documentation fixes.

### PR DESCRIPTION
**Pull request by tomaszmichalak**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,7 +24,7 @@ technologies and release cycles.
 It combines **asynchronous programming principles** and **message-driven architecture** providing **a scalable
 platform** for modern sites.
 
-**Knot.x** connects all of the above **in a unified customer experience**.
+**Knot.x** connects all of the above **into a unified customer experience**.
```
Comment (toniedzwiedz) [inline README.md]: I don't think you can _connect something **into** something_. Both the old and the new version sound strange. How about `**Knot.x** connects all of the above **to create a unified customer experience**.`

File: README.md [hunk 2]
```diff
@@ -84,11 +84,11 @@ section of this documentation.
...
   alt="Knot.x Extensions"/>
 
-You need to implement custom authentication mechanism for your site and then integrate with service
-talking with its own custom protocol. **Knot.x** is a fully modular platform with very flexible extension
+You need to implement custom authentication mechanism for your site and then integrate with service which 
+uses its own custom protocol. **Knot.x** is a fully modular platform with very flexible extension
 points that we call [Knots](https://github.com/Cognifide/knotx/wiki/Knot) and [Adapters](https://github.com/Cognifide/knotx/wiki/Adapter).
 
-Those extension points communicates with Knot.x Core using a very performant Event Bus so you can
+Those extension points communicate with Knot.x Core using a efficient Event Bus so you can
```
Comment (toniedzwiedz) [inline README.md]: _using **an** efficient Event Bus_

File: README.md [hunk 3]
```diff
@@ -84,11 +84,11 @@ section of this documentation.
   src="https://github.com/Cognifide/knotx/blob/master/documentation/src/main/wiki/assets/knotx-intro-extensions.png?raw=true"
   alt="Knot.x Extensions"/>
 
-You need to implement custom authentication mechanism for your site and then integrate with service
-talking with its own custom protocol. **Knot.x** is a fully modular platform with very flexible extension
+You need to implement custom authentication mechanism for your site and then integrate with service which
```
Comment (toniedzwiedz) [inline README.md]: _You need to implement **a** custom authentication mechanism for your site and then integrate with **a** service_

File: README.md [hunk 4]
```diff
@@ -67,7 +67,7 @@ section of the **Knot.x** documentation.
   src="https://github.com/Cognifide/knotx/blob/master/documentation/src/main/wiki/assets/knotx-intro-prototyping.png?raw=true"
   alt="Knot.x Prototyping"/>
 
-A potential client asked you to prepare a demo presenting the capabilities of a new site. The client
+A potential client asks you to prepare a demo presenting capabilities of a new site. The client
```
Comment (toniedzwiedz) [inline README.md]: _A potential client asks you to prepare a demo presenting **the** capabilities of a new site._

You shouldn't have removed **the**

File: README.md [hunk 5]
```diff
@@ -54,9 +54,9 @@ section to find out more about this topic.
   alt="Knot.x Forms"/>
 
 Every site contains more or less complicated forms. **Knot.x** supports simple and multi-step forms.
-It handles submission errors, form validation and redirects to success pages.
+It handles submission errors, form validations and redirects to success pages.
```
Comment (toniedzwiedz) [inline README.md]: Why change this?

Comment (tomaszmichalak) [inline README.md]: plural, singular, plural

File: README.md (hunk 1 above)
Comment (tomaszmichalak) [inline README.md]: nice

File: README.md (hunk 4 above)
Comment (tomaszmichalak) [inline README.md]: yeah, I though about this change, maybe you are right

**Pull request merged**